src/System.ServiceModel.Primitives/tests/Description/OperationBehaviorTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.Threading.Tasks;
+using Xunit;
+
+public static class OperationBehaviorTest
+{
+    [Fact]
+    [ActiveIssue(280)]
+    public static void IOperationBehavior_Methods_AreCalled()
```

Comment:
I know you mentioned this doesn't work and have opened #280.  Does this code work on the desktop?